src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -26,10 +27,7 @@ export const NewsletterSignup = ({
   <Section className="newsletter-signup">
     <Grid>
       <GridCell column={1} columnCount={1}>
-        <h3
-          className="newsletter-signup__title"
-          id={title.replace(/\s+/g, '-').toLowerCase()}
-        >
+        <h3 className="newsletter-signup__title" id={inpageId}>
```

Comment:
I presume the purpose of this to allow users to access newsletter signup from inpage nav?

If so you should take a look at section title to see how header overlap is dealt with (basically `.cc-section__title-anchor` positions a child span above the title to effectively position the title text itself underneath the header)
https://github.com/wellcometrust/corporate-components/blob/98259b8cd4211b44ce569fec13c5fd6065000542/src/components/SectionTitle/SectionTitle.tsx#L18

Reply:
It's not for inpage nav, just need to have id so it can be linked from links within the page text.

Reply:
ok

the same would apply though I think, since this is all based around standard HTML anchor functionality

Reply:
I'm bit concerned about what affect will it have on styling when I change h3 to span.

Reply:
NO, don't do that

The fix is to insert a child span _inside_ the H3

Reply:
SectionTitle markup in full - https://github.com/wellcometrust/corporate-components/blob/98259b8cd4211b44ce569fec13c5fd6065000542/src/components/SectionTitle/SectionTitle.tsx#L16-L21

Reply:
I see, ok i have made an update.